### ISSUE TITLE
tracking performance

### ISSUE THREAD

**Issue by kaczmarj** [kaczmarj/nobrainer-tf2#7]
the purpose of this issue is to track performance of models.

unet
----
- using 4 1080ti gpus, the default nobrainer 3d unet can be trained on input of (8, 128, 128, 128, 1) in about 1.5 seconds. at this rate, it trains an epoch of 99 256**3 volumes in circa 151 seconds. this is using jaccard loss and calculating dice similarity at every step. there is a cost in startup time to training across multiple gpus, but it is minimal when considering how long training can take.
- i repeated the above but with batch normalization. i had to reduce the batch size to 4 to avoid resource exhaustion. an epoch took 251 seconds (which is about 2.5 seconds per volume).
- i repeated the first bullet (unet without batchnorm) with augmetation. the batch size was 8. an epoch took 321 seconds (which is about 3.24 seconds per volume). augmentation was applied to every volume.
